shared-scripts.sh

```diff
@@ -235,18 +244,32 @@ function _setupE2ETestsLinux {
     echo "Setup E2E Tests Linux"
     loadCacheFromLinuxBuildJob
     loadCache verdaccio-cache $CODEBUILD_SRC_DIR/../verdaccio-cache
-    _installCLIFromLocalRegistry  
+    _installCLIFromLocalRegistry
+    _loadTestAccountCredentials
+    _setShell
+}
+
+function _setupE2ETestsWindows {
+    echo "Setup E2E Tests Windows"
+    loadCacheFromWindowsBuildJob
+    _installCLIFromLocalRegistry windows
     _loadTestAccountCredentials
     _setShell
 }
 
+
 function _runE2ETestsLinux {
     echo "RUN E2E Tests Linux"
     retry runE2eTest
 }
 
+function _runE2ETestsWindows {
+    echo "RUN E2E Tests Windows"
+    retry runE2eTest
+}
+
 function _scanArtifacts {
-    if ! yarn ts-node .codebuild/scripts/scan_artifacts.ts; then
```

Comment:
Get an error when using yarn over npx/npm. It looks like there are some workarounds, but it is easier to just use npx/npm here because there will be no functional difference to the scripts.

```
error Couldn't find the binary C:\codebuild\tmp\output\src2004\src\github.com\aws-amplify\amplify-codegen\node_modules\.bin\ts-node
```